syntax="proto3";

package user;

// Define the User message
message User {
    string id = 1;
    string first_name = 2;
    string last_name = 3;
    string password = 4;
    string email = 5;
    bool is_verified = 6;
    string verified_at = 7; // ISO 8601 format
    string updated_at = 8; // ISO 8601 format
    string created_at = 9; // ISO 8601 format
}

// Define the UserToken message
message UserToken {
    string id = 1;
    string user_id = 2;
    string token = 3;
    string created_at = 4; // ISO 8601 format
    string expired_at = 5; // ISO 8601 format
}

